com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass-Creating.md

```diff
@@ -159,6 +159,83 @@ However, not all materials are supported by every injection point in a draw rend
+| Layer Mask                        |Select the GameObject layer that this Custom Pass applies to.                                                 |
+| Overrides           |                                                              |
+|Override Depth                     |Enable this checkbox to override the depth render state in the materials of the rendered GameObjects.<br/><br/> This allows you to replace the default **Depth Test** value, and write the depth using custom values.                                                              |
+|Depth Test                     | When Unity renders an GameObjects, it uses the **Depth Test** value to check if it is behind another object. To do this, Unity tests the z-value (the depth) of a given GameObject’s pixel, and compares it against a value stored in the depth buffer. <br/>By default, **Depth Test** is set to **Less Equal**, allowing the original object to appear in front of the object it is tested against. Use the drop-down to select the comparison method to use for the depth test. Each comparison method changes how the Shader renders:<br/><br/>**Disabled**: Do not perform a depth test.<br/>**Never**: The depth test never passes.<br/>**Less**: The depth test passes if the pixel's z-value is less than the stored value.<br/>**Equal**: The depth test passes if the pixel's z-value is equal to the stored value.<br/>**Less Equa**l: The depth test passes if the pixel's z-value is less than or equal than the Z-buffers value. This renders the tested pixel in front of the other.<br/>**Greater**: The depth test passes if the pixel's z-value is greater than the stored value.<br/>**Not Equa**l: The depth test passes if the pixel's z-value is not equal to the stored value.<br/>**Greater Equa**l: The depth test passes if the pixel's z-value is greater than or equal to the stored value.<br/>**Always**: The depth test always passes, there is no comparison to the stored value.This setting only appears when you enable **Override Depth**.      |
+|Write Depth                     |Enable **Write Depth** to instruct Unity to write depth values for GameObjects that use this material. <br/>Disable it if you do not want Unity to write depth values for each GameObject.                                                              |
+|  Sorting                     | Select the way Unity sorts the GameObjects in your scene before it renders them.<br/><br/>For more information, see [Sorting criteria](https://docs.unity3d.com/ScriptReference/Rendering.SortingCriteria.html).                                          |
+
+### Generating Object IDs for a custom pass
+
+HDRP generates Object IDs before executing an Object ID custom pass.
+
+If you use a script to generate new GameObjects procedurally, call `AssignObjectIDs` to generate new Object IDs.
+
+The following script example uses the default `AssignObjectIDs`  implementation that assigns Object ID colors incrementally:
```

Comment:
Small suggestion: 
"The following script example **uses** the default" ---> "The following script example **shows** the default"

The script shows the default implementation of the AssignObjectIDs method (so users know what to expect), it's not "using" this method.

Reply:
Thanks for this clarification, I'll make the change :)